sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StatefulAggregate.scala

```diff
@@ -54,6 +55,18 @@ trait StatefulOperator extends SparkPlan {
   }
 }
 
+trait StateStoreReader extends StatefulOperator {
```

Comment:
This files should probably be renamed from StatefulAggregation to StatefulOperations.